packages/@aws-cdk/cfnspec/build-tools/create-missing-libraries.ts

```diff
@@ -231,6 +232,7 @@ async function main() {
       'tsconfig.json',
       '',
       '.eslintrc.js',
+      'jest.config.js',
```

Comment:
Github doesn't let me comment on a line that is not changing in this PR, but this needs to be fixed up - 

https://github.com/aws/aws-cdk/blob/7dd5ca0356f81825f0d0338696e982db45765e1d/packages/%40aws-cdk/cfnspec/build-tools/create-missing-libraries.ts#L166

Reply:
Good catch, done!